.circleci/upload_data_example.yml

```diff
@@ -51,15 +58,15 @@ data_info:
     cases: 37846
     controls: 318237
     # accepted data types include: gwas, twas, eqtl, sqtl, pqtl, tad
-    data_type: twas
+    data_type: gwas
```

Comment:
`build` should be added here too, because according to the latest PR in `cimr`, it is a required field.

Reply:
thank you! I am updating the document with new required fields and replacing this example file with others in the submitted dir.